### PR TITLE
Added ex. list for lesson 4

### PR DESCRIPTION
**Pull request by mateiidavid**

### CODE REVIEW

File: docs/exercises/exercise-list-ch-4.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Exercise list _Chapter 4_
```
Comment (Ben-Woolley) [inline docs/exercises/exercise-list-ch-4.md:1]: Needs to be linked to the lesson page, and there are exercises on that page that you could add to this one

Comment (mateiidavid) [reply docs/exercises/exercise-list-ch-4.md:1]: Coolio will do it asap, probably later on when I get home

**Comment by mateiidavid** [comment]
# todo: 

- remove implementation from BankAccount (used it to write tests)

**Comment by Ben-Woolley** [comment]
Do we want to clear out the docs (of pokemon) at the same time or in a different PR?

**Pull request merged**